packages/eslint-plugin/lib/rules/no-nested-await.js

```diff
@@ -45,6 +45,24 @@ module.exports = {
     supported: true || CodePathAnalyzer !== null,
   },
   create(context) {
+    const isValidParent = parent => {
```

Comment:
Maybe a more descriptive name than "valid"?

```suggestion
    const isAwaitableParent = parent => {
```